docs/products/kafka/kafka-connect/howto/gcp-bigquery-sink.rst

```diff
@@ -0,0 +1,245 @@
+To setup an Google BigQuery sink connector, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to follow the steps :doc:`to prepare the GCP account <gcp-bigquery-sink-prereq>` and collect the following information about the target BigQuery upfront:
+
+* ``GCP_PROJECT_NAME``: The GCP project name where the target Google BigQuery is located
+* ``GCP_SERVICE_KEY``: A valid GCP service account key for the ``GCP_PROJECT_NAME``. To create the project key review the :ref:`dedicated document <gcp-bigquery-sink-connector-google-account>`
+
+.. Warning::
+
+    The GCP BigQuery sink connector accepts the ``GCP_SERVICE_KEY`` JSON service key as string, therefore all  ``"`` symbols within it must be escaped ``\"``.
+
+    The ``GCP_SERVICE_KEY`` parameter should be in the format ``{\"type\": \"service_account\",\"project_id\": \"XXXXXX\", ...}``
+
+    Additionally, any ``\n`` symbols contained in the ``private_key`` field need to be escaped (by substituting with ``\\n``)
+
+* ``BIGQUERY_DATASET_NAME``: The BigQuery dataset name, as defined in the :ref:`dedicated pre-requisite step <gcp-bigquery-sink-connector-bigquery-dataset>`
+* ``TOPIC_LIST``: The list of topics to sink divided by comma
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    The ``SCHEMA_REGISTRY`` related parameters are available in the Aiven for Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
+
+    As of version 3.0, Aiven for Apache Kafka no longer supports Confluent Schema Registry. For more information, read `the article describing the replacement, Karapace <https://help.aiven.io/en/articles/5651983>`_
+
+Setup an Google BigQuery sink connector with Aiven Console
+----------------------------------------------------------
+
+The following example demonstrates how to setup a Google BigQuery sink connector for Apache Kafka using the `Aiven Console <https://console.aiven.io/>`_.
+
+Define a Kafka Connect configuration file
+'''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``bigquery_sink.json``) with the following content:
+
+.. code-block:: json
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class": "com.wepay.kafka.connect.bigquery.BigQuerySinkConnector",
+        "topics": "TOPIC_LIST",
+        "project": "GCP_PROJECT_NAME",
+        "datasets": ".*=BIGQUERY_DATASET_NAME",
```

Comment:
There is also the `defaultDataset` option

Reply:
Fixed, since the `datasets` is the old way to reference the parameter (as per https://docs.confluent.io/kafka-connect-bigquery/current/kafka_connect_bigquery_config.html)